libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -367,7 +367,7 @@ bool AP_Arming::gps_checks(bool report)
     }
 
     // check GPSs are within 50m of each other and that blending is healthy
-    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_GPS_CONFIG)) {
+    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_GPS)) {
```

Comment:
I was wondering on this one, I noted that it was config the other day, which I thought was a bit weird, but I also thought there might have been a reason it was not put in as part of the normal GPS checks?